<?xml version="1.0"?>
<launch>
    <arg name="mesh_file" default="box_base"/>
    <arg name="meshes_pkg" default="$(optenv MESHES_PKG component_sorting_description)" />
    <arg name="mesh_name" default="$(eval find(meshes_pkg) + '/meshes/' + mesh_file)"/>

    <arg name="mesh_position_x" default="1"/>
    <arg name="mesh_position_y" default="1"/>
    <arg name="mesh_position_z" default="1"/>
    <arg name="object_id" default="box"/>

    <!--node name="publish_collision_object" pkg="component_sorting" type="publish_collision_object_node" output="screen" args="$(arg mesh_name)"-->
    <node name="publish_collision_object" pkg="component_sorting" type="publish_collision_object_node" output="screen">
        <param name="mesh_name" value="$(arg mesh_file)"/>
        <param name="mesh_position_x" value="$(arg mesh_position_x)"/>
        <param name="mesh_position_y" value="$(arg mesh_position_y)"/>
        <param name="mesh_position_z" value="$(arg mesh_position_z)"/>
        <param name="object_id" value="$(arg object_id)"/>
    </node>
</launch>